### DISCUSSION TITLE
Inconsistent data retentions in Kibana

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#7619] [Unsupported Versions] [answered]
So im trying to build some dashboards in kibana to show stats for the previous month. However, I'm noticing some inconsistencies with how far data is retained. If I go by the alert category, Suri and Wazuh/Ossec only seem to go back last 30 days from the current date but playbooks and Zeek go back to around 90 days. From my understanding looking at the lifecycle policies in kibana indices are kept in a "hot" state and are rolled up once they hit 30 days old and then they are "deleted" after 90 days. i could be missing something but why are some only showing metrics on 30 days and others 90+ days when I thought it was 90 days across the board. Is there a way to change the retention of the suri/Wazuh logs to show more than just last 30 days?

**Comment by dougburks** [comment, marked as answer]
Check your index close settings in the global pillar:
https://docs.securityonion.net/en/2.3/elasticsearch.html#pillar-files
